api/spec/requests/spree/api/v2/storefront/account/addresses_spec.rb

```diff
@@ -67,6 +68,23 @@
       end
     end
 
+    context 'when address from countries that are supported in the current store' do
+      let!(:country) { create(:country, name: 'France') }
+      let!(:zone) { create(:zone, name: 'EU_VAT', countries: [country], kind: 'country') }
+      let!(:eu_address) { create(:address, user_id: user.id, country: country) }
```

Comment:
@v10110 let's also create another address for this user from a country that isn't supported by this store to test if it doesn't show up